main.js

```diff
@@ -0,0 +1,147 @@
+const CHANGE_USER_DATA = (obj) => {
+    USER_1.city = "Kherson";
+    console.log(USER_1)
+}
+CHANGE_USER_DATA(USER_1)
+
+// 4. Створити об'єкт cтудента - name, surname, age, course, city, greeting, addHomework. greeting - метод, котрий виводить повідомлення через console.log('Hi. everyone!'). addHomework - метод, котрий виводить повідомлення через console.log('Sending my howework... Please, wait')
+const STUDENT = {
+    name: "Stas",
+    surname: "Kryzhanivsky",
+    age: 37,
+    course: 4,
+    city: "Odessa",
+    greeting: function() {
+        console.log('Hi. everyone!');
+    },
+    addHomework: function() {
+        console.log('Sending my howework... Please, wait')
+    },
+}
+console.log(STUDENT);
+console.log(STUDENT.greeting);
+console.log(STUDENT.addHomework);
+
+
+// 5. Cтворити функцію isEmpty, яка повертає true, якщо об’єкт не має властивостей (порожній), інакше false.
+const IS_EMPTY = (obj) => {
+    for (let key in obj) {
+        return "false";
+    } 
+        return "true";
+    }
+
+const OBJECT = {};
+const OBJECT_2 = {
+    name: "Anna",
+    age: 30
+};
+
+console.log(IS_EMPTY(OBJECT)); // true
+console.log(IS_EMPTY(OBJECT_2)); // false
+
+// 6. Створити об’єкт для зберігання виручки команди продавців, наприклад: {Taras: 2000, Marta: 10 Ivan: 1200, Petro: 400, Roma: 366, Alina: 829}
+const INCOME_OF_THE_TEAM = {
+    Taras: 2000, 
+    Marta: 10,
+    Ivan: 1200, 
+    Petro: 400, 
+    Roma: 366, 
+    Alina: 829
+}
+
+
+// 7. Створити фукнцію, яка працює з цим обʼєктом та обчислює суму всіх виручок та виводить результат через сonsole.log
+const TOTAL_INCOME = (obj) => {
+    let sum = 0;
+    for (let key in obj) {
+        sum += obj[key];
+    }
+    console.log(sum);
+}
+
+TOTAL_INCOME(INCOME_OF_THE_TEAM);
+
+
+// 8. Створити фукнцію, яка працює з цим обʼєктом та яка знаходить продавця з найменшою виручкою та виводить результат через сonsole.log у зрозумілому форматі
+const GET_MIN_VALUE = (obj) => {
+    const MIN_VALUE = Math.min.apply(null, Object.values(obj));
+    console.log(`${Object.keys(obj).filter(k => obj[k] === MIN_VALUE)} got the smallest income.`);
+}
+
+GET_MIN_VALUE(INCOME_OF_THE_TEAM);
+
+// 9. Створити фукнцію, яка знаходить продавця з найбільшою виручкою та виводить результат через сonsole.log у зрозумілому форматі
+const GET_MAX_VALUE = (obj) => {
+    const MAX_VALUE = Math.max.apply(null, Object.values(obj));
+    console.log(`${Object.keys(obj).filter(k => obj[k] === MAX_VALUE)} got the biggest income.`);
+}
+
+GET_MAX_VALUE(INCOME_OF_THE_TEAM);
+
+// 10. Створити фукнцію, яка випадковим чином вибирає продавця місяця та виводить привітання цьому працівнику через сonsole.log у зрозумілому форматі
+const GET_RANDOM_SELLER = (obj) => {
+    const RANDOM_SELLER = Math.floor(Math.random() * (Object.keys(obj).length) + 0);
```

Comment:
It can be simplified 
```suggestion
    const RANDOM_SELLER = Math.floor(Math.random() * (Object.keys(obj).length));
```